dashboards/availability_exporter.json

```diff
@@ -0,0 +1,510 @@
+              "axisSoftMin": 0,
+              "barAlignment": 0,
+              "drawStyle": "line",
+              "fillOpacity": 6,
+              "gradientMode": "opacity",
+              "hideFrom": {
+                "legend": false,
+                "tooltip": false,
+                "viz": false
+              },
+              "insertNulls": false,
+              "lineInterpolation": "linear",
+              "lineStyle": {
+                "fill": "solid"
+              },
+              "lineWidth": 1,
+              "pointSize": 6,
+              "scaleDistribution": {
+                "type": "linear"
+              },
+              "showPoints": "auto",
+              "spanNulls": false,
+              "stacking": {
+                "group": "A",
+                "mode": "none"
+              },
+              "thresholdsStyle": {
+                "mode": "off"
+              }
+            },
+            "mappings": [],
+            "thresholds": {
+              "mode": "absolute",
+              "steps": [
+                {
+                  "color": "green",
+                  "value": null
+                },
+                {
+                  "color": "red",
+                  "value": 80
+                }
+              ]
+            },
+            "unit": "percent"
+          },
+          "overrides": []
+        },
+        "gridPos": {
+          "h": 12,
+          "w": 24,
+          "x": 0,
+          "y": 7
+        },
+        "id": 18,
+        "options": {
+          "legend": {
+            "calcs": [],
+            "displayMode": "table",
+            "placement": "bottom",
+            "showLegend": true,
+            "sortBy": "Name",
+            "sortDesc": false
+          },
+          "tooltip": {
+            "mode": "single",
+            "sort": "none"
+          }
+        },
+        "targets": [
+          {
+            "datasource": {
+              "type": "prometheus",
+              "uid": "PDCCF087A30F0737B"
+            },
+            "editorMode": "code",
+            "exemplar": false,
+            "expr": "(group by (service, check, source_cluster) (count_over_time((konflux_up{service=~\"$service\", check=~\"$check\"} == 1)[$__range:])) / group by (service, check, source_cluster) (count_over_time(konflux_up{service=~\"$service\", check=~\"$check\"}[$__range:])) ) * 100",
+            "instant": false,
+            "legendFormat": "{{source_cluster}} - {{service}} - {{check}} ",
+            "range": true,
+            "refId": "A"
+          }
+        ],
+        "title": "Individual availability probes",
+        "type": "timeseries"
+      }
+    ],
+    "refresh": "",
+    "schemaVersion": 39,
+    "tags": [],
+    "templating": {
+      "list": [
+        {
+          "current": {
+            "selected": true,
+            "text": "rhtap-observatorium-production",
+            "value": "P22466E8E7855F1E0"
+          },
+          "hide": 0,
+          "includeAll": true,
```

Comment:
I think we can remove the `All` selection for the datasource. I doubt many would find it useful. It doesn't seem to be working anyway.

Reply:
@amisstea I updated it with the suggestions that you requested. PTAL